.gitlab/custom-jobs-and-variables.yml

```diff
@@ -56,7 +56,7 @@ variables:
 # Lassen and Butte use a different job scheduler (spectrum lsf) that does not
 # allow pre-allocation the same way slurm does.
 # Arguments for job level allocation
-  LASSEN_JOB_ALLOC: "1 -W 30 -q pci"
+  LASSEN_JOB_ALLOC: "1 -W 40 -q pci"
```

Comment:
uh oh... are we hitting limits?

Reply:
Ya now there are 6 reduce policies we're testing for cuda.